telegram/user.py

```diff
@@ -189,6 +189,45 @@ def mention_html(self, name: str = None) -> str:
             return util_mention_html(self.id, name)
         return util_mention_html(self.id, self.full_name)
 
+    def pin_message(self, *args: Any, **kwargs: Any) -> bool:
```

Comment:
why does `User` get `un/pin_message` shortcuts and `Chat` doesn't? :D Tbh I don't really see a big use case for `User/Chat.un/pin_message`, but having doesn't hurt. In any case, either both should have them or none of them :)

Reply:
Yea my bad, I was being lazy. Changed in the next commit